src/dotnet/commands/dotnet-publish/Program.cs

```diff
@@ -69,7 +69,7 @@ public static int Run(string[] args)
             }
             catch (Exception ex)
             {
-                Reporter.Error.WriteLine(ex.Message.Red());
+                Reporter.Error.WriteLine(ex.ToString().Red());
```

Comment:
I'm not sure we want to always print the entire to string to the user.  We don't for other commands.